### PR TITLE
docs: add troubleshooting for argocd problem

### PR DESCRIPTION
**Pull request by njtran**
Fixes # 

**Description**

**How was this change tested?**

*

**Does this change impact docs?**
- [x] Yes, PR includes docs updates
- [ ] Yes, issue opened: # 
- [ ] No

**Release Note**

```release-note

```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: website/content/en/preview/troubleshooting.md [hunk 1]
```diff
@@ -8,6 +8,34 @@ description: >
 
 ## Installation
 
+### Installation Webhook Timeout on v0.19.0+
+
+If you're upgrading from before `v0.19.0` to `v0.19.0+`, are using ArgoCD, and are unable to call your webhook, your ArgoCD sync may be trying to check for webhooks that no longer exist, since the name has changed.
```
Comment (ellistarn) [inline website/content/en/preview/troubleshooting.md:13]: > unable to call your webhook

What's the error message look like?

File: website/content/en/preview/troubleshooting.md [hunk 2]
```diff
@@ -8,6 +8,34 @@ description: >
...
+
+Add the following to the ArgoCD `application.yaml` for Karpenter under the spec object.
```
Comment (ellistarn) [inline website/content/en/preview/troubleshooting.md:15]: I think we can skip this guidance.

File: website/content/en/preview/troubleshooting.md [hunk 3]
```diff
@@ -8,6 +8,34 @@ description: >
...
+```
+
+Once this is done, update the chart and sync. Then delete the stale webhooks with the following:
+```
+kubectl delete mutatingwebhookconfigurations defaulting.webhook.provisioners.karpenter.sh
+kubectl delete validatingwebhookconfiguration validation.webhook.provisioners.karpenter.sh
+```
+After this, sync ArgoCD once again. If this does not work, you can also try to delete the CRDs used in Karpenter and then re-sync.
+```
+kubectl delete crd awsnodetemplates.karpenter.k8s.aws provisioners.karpenter.sh
```
Comment (ellistarn) [inline website/content/en/preview/troubleshooting.md:36]: This one seems like a bad idea to me. Will cascading delete all nodes.

Comment (njtran) [reply website/content/en/preview/troubleshooting.md:36]: good point. Will remove.

**Review by ellistarn — APPROVED** [review]

**Pull request merged**